lib/Loop/Loop.php

```diff
@@ -302,7 +302,7 @@ protected function runStreams(?float $timeout): void
     /**
      * A list of timers, added by setTimeout.
      *
-     * @var array<int, mixed>
+     * @var array<int, array{0:float, 1:callable}>
```

Comment:
another improvement can be to turn some of the properties into `@var list<..>` instead of array

Reply:
See PR #114